### PR TITLE
fix: limit and spinning of search icon

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
make sure you are searching by adding spinner

https://github.com/ansys/ansys-sphinx-theme/assets/104772255/03661c50-56b4-4dbc-a640-ed6eac864a53

### CODE REVIEW

**Comment by germa89** [comment]
Loving it.

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/meilisearch.css [hunk 1]
```diff
@@ -92,23 +92,12 @@ div[data-ds-theme] .meilisearch-autocomplete {
...
-}
-
-.meilisearch-autocomplete::before {
-  content: "\f002";
-  font-family: "Font Awesome 6 Free";
-  position: absolute;
-  left: 8px;
-  top: 50%;
-  transform: translateY(-50%);
-  font-size: 1rem;
```
Comment (germa89) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/meilisearch.css:110]: So the autocomplete  is somewhere else now?

**Review by germa89 — APPROVED** [review]
Looks awesome to meee!

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/meilisearch.css (hunk 1 above)
Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/meilisearch.css:110]: yes, this was for icon using css, added that icon in js.

**Review by MaxJPRey — APPROVED** [review]
LGTM.
Thanks for the gif @Revathyvenugopal162 in the description.

**Pull request merged**